chain/gen/genesis/miners.go

```diff
@@ -271,6 +271,8 @@ func SetupStorageMiners(ctx context.Context, cs *store.ChainStore, sroot cid.Cid
 		return cid.Undef, xerrors.Errorf("mutating state: %w", err)
 	}
 
+	// TODO: Should we re-ConstructState for the reward actor using rawPow as currRealizedPower here?
```

Comment:
Good question, I feel like it is set correctly when we import sectors, but that warrants checking, will open an issue

Reply:
Opened https://github.com/filecoin-project/lotus/issues/2389